addons/delete-others/userscript.js

```diff
@@ -0,0 +1,99 @@
+export default async function ({ addon, console, msg }) {
+  const vm = addon.tab.traps.vm;
+
+  const TYPES = ["sound", "costume"];
+  let deletedItems = [];
+  let target;
+  let shouldChangeRestoreButtonText = false;
+
+  function getRestoreFun(type) {
+    deletedItems.reverse();
+    if (type === "costume") {
+      deletedItems.forEach((costume) => {
+        vm.addCostume(costume.md5, costume, target.id, "");
```

Comment:
I'm getting an error on this line.
`Uncaught TypeError: Cannot read properties of undefined (reading 'indexOf')`

See:
- https://github.com/scratchfoundation/scratch-vm/blob/f638db00447ce13b4a506068e52aef0ab1a9b641/src/import/load-costume.js#L359
- https://github.com/scratchfoundation/scratch-vm/blob/f638db00447ce13b4a506068e52aef0ab1a9b641/src/util/string-util.js#L34

Reply:
The root cause is that `costume.md5` is undefined. I don't know if it's all costumes or only the ones I'm playing with.

Reply:
Suggestion by GarboMuffin

```suggestion
        vm.addCostume(costume.assetId + "." + costume.dataFormat, costume, target.id, "");
```